File: api/code-samples/javascript/klaytnDebug/profilling/writeMemProfile.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));
    
    const file = "mem.profile";

    sdk.debug.writeMemProfile(file, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: api/code-samples/python/debug/profiling/writeMemProfile.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

fileName = "mem.profile"

sdk = OpenSDK(host)
debug_response = sdk.debug.write_mem_profile(fileName)

print(json.loads(debug_response.response.data))
